docs-src/cloud/prerequisites-to-exporting-workflow-history.md

```diff
@@ -0,0 +1,16 @@
+---
+id: prerequisites-to-exporting-workflow-history
+title: Prerequisites
```

Comment:
atm - this seems like the only Export location we support?
It could be "Provision an S3 bucket" or something similar to specify the step -